readme.rst

```diff
@@ -174,6 +174,20 @@ Add custom configuration
 To add custom configurations for your project, see the ``Additions to default configuration`` and ``Additional configuration`` sections in the ``custom_conf.py`` file.
 These can be used to extend or override the common configuration, or to define additional configuration that is not covered by the common ``conf.py`` file.
 
+(Optional) Synchronise GitHub issues to Jira
+--------------------------------------------
+
+If you wish to sync issues from your documentation repository on GitHub to your
+Jira board, configure the `gh-jira-sync-bot <https://github.com/canonical/gh-jira-sync-bot>`_
```

Comment:
This breaks the spelling check - you'll need to add "Jira" to the wordlist. And "gh-jira-sync-bot" fails as well ... maybe easiest to call it "GitHub/Jira sync bot"?

Reply:
Done, thanks!

Reply:
You probably need to change it in this line here as well.